### PR TITLE
fix(alerts): rewriting the alert condition page, deleting extra docs

### PR DESCRIPTION
**Pull request by homelessbirds**
The alert condition page needed a revamp. I merged Configure and Advance Alert conditions and modified the NRQL condition doc.

### CODE REVIEW

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 1]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
 
 Any alert condition built with a synthetic monitor will be a bit different. This is because synthetic monitors allow you to ping your application from multiple locations, which can produce positive or negative results each time the monitor runs. This data can only be presented with a table.
 
-## Troubleshoot [#troubleshoot]
+## Types of conditions [#condition-types]
+
+The primary and recommend condition type is a NRQL alert condition, but there are other types of conditions.  We've included a complete list of our condition types below.
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:382]: Typo `recommended`

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 2]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+    id="infrastructure"
+    title="Infrastructure conditions"
+  >
+    You can [create conditions for your resources](/docs/infrastructure/new-relic-infrastructure/configuration/infrastructure-alerts-add-edit-or-view-host-alert-information) directly from our infrastructure monitoring UI.
+
+    For example, if you want to be notified when we have stopped receiving data from an infrastructure agent, use the [host not reporting](/docs/infrastructure/new-relic-infrastructure/configuration/create-infrastructure-host-not-reporting-condition) condition type. This allows you to dynamically alert on filtered groups of hosts and configure the time window from 5 to 60 minutes.
+  </Collapser>
+</CollapserGroup>
+
+## Apdex and response time conditions [#alert_response]
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:491]: Should this be in a `collapsergroup` as well?

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 3]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+
+You can open incidents and send notifications for response times. However, [Apdex scores](/docs/apm/new-relic-apm/apdex/view-your-apdex-score) are almost always more meaningful and provide a better reflection of application performance. For example, average response times can be skewed by outliers, while the Apdex score gives a more accurate assessment of acceptable response time rates that your users experience.
+
+Here's a quick reference for maintaining conditions. This includes the condition's entities (targets), Warning (yellow) and Critical (red) thresholds, and runbook URL.
+
+## Add conditions to a policy [#add-conditions]
+
+To [add more conditions to a policy](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions):
+1.  Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Policies**</DNT>
+2. Delect a policy.
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:501]: Typo

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 4]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+3. From the <DNT>**Copy alert condition**</DNT>, search or scroll the list to select the policy where you want to add this condition.
+4. Optional: Change the condition's name if necessary.
+5. Optional: Click the toggle switch to <DNT>**Enable on save**</DNT>
+6. Select <DNT>**Copy condition**</DNT>.
+
+    By default, the selected alert policy will add the copied condition in the **Disabled** state. Follow standard procedures to [add](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) or copy more conditions to the alert policy, and then [**Enable** the condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#disable-conditions) as needed. Additionally, the new condition will not copy any [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-via-ui-api) added to the original condition.
+
+## Configure an existing condition [#configure-condition]
+
+To change a condition, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT> and click a condition.
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:524]: Remove `and click a condition`

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 5]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+
+* To [change the condition's name](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions#rename-condition) select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+* To update the values for the required Critical (red) or optional Warning (yellow) [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list..
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:527]: remove `required` and `optional`. This info is outdated. A condition can have a warning threshold only.

Comment (akane0915) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:527]: Also remove double period at the end.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 6]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+* To update the condition's [runbook URL](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/provide-runbook-instructions-alerts), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+
+## Enable/disable a condition [#disable-conditions]
+
+You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:532]: What does this mean? `...and the policy will continue to apply`.  This doesn't make sense to me.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 7]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT>. Then, from the list of <DNT>**Alert conditions**</DNT>, use the toggle to enable or disable the condition.
+2. Click the <DNT>**On/Off**</DNT> switch to toggle it.
+
+If you [copy a condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/copy-alert-conditions), it automatically saves it in the new policy as disabled (<DNT>**Off**</DNT>), even if the condition was enabled (<DNT>**On**</DNT>) in the original policy.
+
+## Delete a condition [#delete-conditions]
+
+If a policy has multiple conditions, you can delete any number of them, and the remaining conditions for the policy will continue to apply. To turn a condition off but keep it with the policy, [disable](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#condition-on-off) it. To delete one or more conditions:
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:541]: `and the remaining conditions for the policy will continue to apply.` is it necessary to say this?

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 8]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+2. From the list of <DNT>**Alert conditions**</DNT>, select a condition, then click <DNT>**Delete**</DNT> from the ellipses menu (...).
+
+    <Callout variant="tip">
+      If you don't see the delete button, your account admin may have disabled condition deletion for your organization.
+    </Callout>
+
+## Troubleshoot the alert condition page [#troubleshoot]
 
-If you see an empty signal in the history chart, consider one of the following:
+If you see an empty signal in the history chart on the **Alert Conditions** page, consider one of the following:
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:552]: This should be consistent with the title.  Singular `condition`.

File: src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx [hunk 9]
```diff
@@ -26,96 +26,20 @@ import queriesnrqlSignalLossUi from 'images/queries-nrql_screenshot-full_signal-
...
-  <img
-    title="Create alert condition from the metrics explorer"
-    alt="Create alert condition from the metrics explorer"
-    src={queriesMetricExplorerAnimation}
-  />
+You can also use one of our alert builders:
 
-4. Follow the steps explained in the [Create a NRQL alert condition from a policy](#alert-condition-from-policy) but be aware that step 1 is a bit different because, now, the query is built with the data you've previously configured.
+* Use <DNT>**Write your own query**</DNT> to build alerts from scratch
+* <DNT>**Use guided mode**</DNT>
```
Comment (akane0915) [inline src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx:40]: Consider changing to `Use guided mode to choose from options and have your NRQL query built for you`

**Review by akane0915 — CHANGES_REQUESTED** [review]
See inline comments.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:491]: Thanks!

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:382]: good catch! thanks!

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx [hunk 10]
```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
...
+    title="APM metric alert conditions"
+  >
+    See [APM metric alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/apm-metric-alert-conditions) for tips about creating <InlinePopover type="apm" /> conditions using NRQL.
+  </Collapser>
+
+  <Collapser
+    id="anomaly-conditions"
+    title="Anomaly conditions"
+  >
+    [Anomaly alerting](/docs/alerts/new-relic-alerts/configuring-alert-policies/create-anomaly-alert-conditions) allows you to create conditions that dynamically adjust to changing data and trends, such as weekly or seasonal patterns. This feature is available for [<InlinePopover type="apm" />](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [<InlinePopover type="browser" />](/docs/browser/new-relic-browser/getting-started/new-relic-browser) apps, as well as [NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries).
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:403]: It's probably very old! I got this from Sean Henry almost a year ago.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:501]: good catch! Thanks!

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:524]: Thanks!

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 7 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:541]: You're right, it's not necessary. I am deleting this.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 8 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:552]: Fixed!

File: src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx (hunk 9 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx:40]: Good call! I fixed this.

File: src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx:532]: You're right, that's not clear. I have removed it.

**Review by nbaenam — APPROVED** [review]

**Pull request merged**